lib/packages.cf

```diff
@@ -1494,7 +1494,7 @@ body package_method freebsd
       package_changes => "individual";
 
       # Could use rpm for this
-      package_list_command => "/usr/sbin/pkg_info";
+      package_list_command => "/usr/sbin/pkg info";
```

Comment:
When did these commands change? Do we need to account for backward compatibility for the older clients that only have `pkg_info`?